source/scss/variables/_color-variables.scss

```diff
@@ -5,36 +5,73 @@ $gray-400: #929497;
 $gray-600: #666666;
 $gray-800: #424242;
```

Comment:
Please remove old colors. This is about updating, not adding more colors. :)
We should be able to see only those colors, which are shown in the Invision file.